lib/tasks/dummy.rake

```diff
@@ -0,0 +1,23 @@
+namespace :dummy do
+  desc 'create dummy data'
+  task create: :environment do
+    u = User.create!(name: 'Pocke')
+
+    cat_with_source = Category.create!(name: 'Vim', description: 'for vim user')
+    source = PostSource.create!(
+      name: 'Vim',
+      user: u,
+      category: cat_with_source,
+    )
+
+    Subscribe.create!(
+      user: u,
+      category: cat_with_source,
+    )
+
+    Post.create!(
+      value: "Vim is awesome!",
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)